test/src/v1/types/geocoder.ts

```diff
@@ -0,0 +1,54 @@
+export type GeocoderFeatureResponseType = Array<{
+  type: string;
+  geometry: {
+    type: string;
+    coordinates: Array<number>;
+  };
+  properties: {
+    id: string;
+    gid: string;
+    layer: string;
+    source: string;
+    source_id: string;
+    name: string;
+    street: string;
+    distance: 1.895;
```

Comment:
This looks weirdly specific for a type 🤔 `number` instead?

Reply:
Indeed!